diff --git a/dist/index.d.ts b/dist/index.d.ts
index 783754d825bb188cc7dd77909018674870b149f5..3ab0280f494b1d6975518194f5da62d7f2daee3c 100644
--- a/dist/index.d.ts
+++ b/dist/index.d.ts
@@ -140,4 +140,4 @@ interface ESLintSukkaOptions {
 }
 declare const sukka: (options?: ESLintSukkaOptions, ...userConfig: FlatESLintConfigItem[]) => Promise<FlatESLintConfigItem[]>;
 
-export { ignores, sukka };
+export { ignores, sukka, ESLintSukkaOptions };
